content/docs/command-reference/exp/show.md

```diff
@@ -235,8 +245,34 @@ $ dvc exp show --all-commits --include-params=featurize --sort-by=auc --sort-ord
 └───────────────────────┴──────────────┴─────────┴────────────────────────┴──────────────────┘
 ```
 
-Note that in the final example, Git commits remain in chronological order. The
+Note that in this example, Git commits remain in chronological order. The
 sorting only applies to experiment groups (sharing a parent commit).
 
+## Example: Parallel coordinates plot (PCP)
+
+To generate an interactive PCP based on the experiments and their parameters:
+
+```dvc
+$ dvc exp show --html
+```
+
+![](/img/ref_parallel_coordinates.png) _Parallel Coordinates Plot_
+
+Use `--sort-by` to customize the colors of the lines:
```

Comment:
Instead of customize, we should be saying something like "the intensity of the background color is determined by the value of the column you select with `--sort-by`."